Gemfile

```diff
@@ -9,10 +9,9 @@ gem 'rails', '~> 5.2.1'
 gem 'bcrypt', '3.1.12'
 
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
-
-gem "bootstrap_form", "~> 2.7"
-gem "bootstrap-sass"
+gem 'sassc-rails'
```

Comment:
`sass-rails` is deprecated, they recommend using `sassc-rails` now

https://www.google.com/search?q=sass-rails+deprecated&oq=sass-rails+depre&aqs=chrome.0.0j69i57.2806j0j1&sourceid=chrome&ie=UTF-8